### PR TITLE
GUI 1st Commit

### PR DESCRIPTION
**Pull request by debajit15kgp**
Merging the first commit and will update with more controls with coming time.

### CODE REVIEW

**Comment by saurabhkgp21** [comment]
@debajit15kgp can you post a screenshot of GUI here?

**Comment by thebhatman** [comment]
Let this PR have changes related to GUI only. Move the commit of random opponents and plays to another branch and send a new PR for Plays.

File: team.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+p1.start()
+p2.start()
+p3.start()
+p4.start()
+p5.start()
+p1.join()
+p2.join()
+p3.join()
+p4.join()
+p5.join()
```
Comment (MayankB11) [inline team.py]: Make them an array of processes

Comment (saurabhkgp21) [reply team.py]: @MayankB11 so much time to review the PR

**Pull request closed without merging**